src/components/savings-plus/savings-plus-manage.vue

```diff
@@ -0,0 +1,151 @@
+import { mapGetters, mapState } from 'vuex';
+
+import dayjs from 'dayjs';
+
+import { SavingsMonthBalanceItem } from '@/services/mover';
+import { fromWei, multiply } from '@/utils/bigmath';
+import { formatToNative, getSignIfNeeded } from '@/utils/format';
+import { dateFromExplicitPair } from '@/utils/time';
+import { getUSDCAssetData } from '@/wallet/references/data';
+
+import { BarChart } from '@/components/charts';
+import { SecondaryPage, SecondaryPageHeader } from '@/components/layout';
+import { StatementsNavList } from '@/components/statements-nav-list';
+
+export default Vue.extend({
+  name: 'SavingsPlusManage',
+  components: {
+    BarChart,
+    SecondaryPageHeader,
+    SecondaryPage,
+    StatementsNavList
+  },
+  data() {
+    return {
+      monthName: dayjs().format('MMMM'),
+      selectedItem: undefined as SavingsMonthBalanceItem | undefined
+    };
+  },
+  computed: {
+    ...mapState({ colors: 'colors' }),
+    ...mapState('account', {
+      networkInfo: 'networkInfo'
+    }),
+    ...mapState('savingsPlus', {
+      isInfoLoading: 'isInfoLoading'
+    }),
+    ...mapGetters('savingsPlus', {
+      info: 'info',
+      monthStatsOptions: 'monthStatsOptions',
+      infoBalanceNative: 'infoBalanceNative',
+      infoEarnedThisMonthNative: 'infoEarnedThisMonthNative',
+      usdcNativePrice: 'usdcNativePrice',
+      savingsMonthStatsOptions: 'savingsMonthStatsOptions'
+    }),
+    savingsBalance(): string {
+      return `$${formatToNative(this.infoBalanceNative)}`;
+    },
+    chartDataSource(): Array<SavingsMonthBalanceItem> {
+      return this.info !== undefined ? this.info.last12MonthsBalances : [];
+    },
+    selectedItemPrefix(): string {
+      if (this.selectedItem === undefined) {
+        return this.$t('savings.lblEarnedRelativeMonthlyChange') as string;
+      }
+
+      const now = dayjs();
+      if (
+        this.selectedItem.month - 1 == now.get('month') &&
+        this.selectedItem.year == now.get('year')
+      ) {
+        return this.$t('savings.lblEarnedRelativeMonthlyChange') as string;
+      }
+
+      return this.$t(
+        'savings.lblEarnedRelativeMonthlyChangeExtendedMonthOnlyPrefix',
+        {
+          date: dateFromExplicitPair(
+            this.selectedItem.year,
+            this.selectedItem.month
+          ).format('MMMM')
+        }
+      ) as string;
+    },
+    selectedItemValue(): string {
+      if (this.selectedItem === undefined) {
+        return this.formatSelectedItemValue(this.infoEarnedThisMonthNative);
+      }
+
+      const now = dayjs();
+      if (
+        this.selectedItem.month - 1 == now.get('month') &&
+        this.selectedItem.year == now.get('year')
+      ) {
+        return this.formatSelectedItemValue(this.infoEarnedThisMonthNative);
+      }
+
+      return this.formatSelectedItemValue(
+        multiply(
+          fromWei(
+            this.selectedItem.earned,
+            getUSDCAssetData(this.networkInfo.network).decimals
```

Comment:
```suggestion
            getUSDCAssetData(this.currentNetwork).decimals
```

`this.currentNetwork` is exposed by global mixin